teams/wg-secure-code.toml

```diff
@@ -21,3 +27,10 @@ zulip-stream = "wg-secure-code"
 
 [[lists]]
 address = "secure-code-wg@rust-lang.org"
+
+[[discord-roles]]
+name = "WG-secure-code"
+color = "#c0c0c0"
+
+[[zulip-groups]]
+name = "WG-secure-code"
```

Comment:
The Zulip channel is lowercase, so let's keep it that way just in case.

Reply:
this style aligns to the others in the repo